apps/tup-ui/src/components/auth/LoginComponent/LoginComponent.tsx

```diff
@@ -14,6 +14,10 @@ type LoginInfo = {
   password: string;
 };
 
+type LoginProps = {
+  className?: string;
+};
+
```

Comment:
ℹ️ So the login form container can be styled as content of its parent component.